### PR TITLE
A container and Top menu bar 

### PR DESCRIPTION
**Pull request by samyakbvs**
Container increases the code quality and also #17 fixed

### CODE REVIEW

**Comment by samyakbvs** [comment]
I won't be able to provide a deployment link but here's a screenshot of the what will the site look like
screen shot 2017-12-08 at 11 03 56 pm
Also, I assure you that links are working perfectly

File: _site/index.html [hunk 1]
```diff
@@ -15,49 +15,24 @@
...
-	</ul>
-	</nav>
-</header>
-<div id="content">
-
-    <div class="page-content">
-      <div class="wrapper">
-        <h1 id="schedule">Schedule</h1>
+<div class = "container" >
+	<ul class="sf-menu"><li class="leaf first no-dhtml "><a href="https://2014.fossasia.org/wrap-up" title="FOSSASIA 2014 Wrap up" id="dhtml_menu-3672">Wrap Up</a></li>
```
Comment (anshumanv) [inline _site/index.html]: Correct indentation of this block please.

**Comment by samyakbvs** [comment]
Is it ready to merge?

File: _site/index.html [hunk 2]
```diff
@@ -15,49 +15,24 @@
...
-	</nav>
-</header>
-<div id="content">
-
-    <div class="page-content">
-      <div class="wrapper">
-        <h1 id="schedule">Schedule</h1>
+    <div class = "container" >
+	<ul class="sf-menu"><li class="leaf first no-dhtml "><a href="https://2014.fossasia.org/wrap-up" title="FOSSASIA 2014 Wrap up" id="dhtml_menu-3672">Wrap Up</a></li>
+<li class="leaf  no-dhtml "><a href="https://2014.fossasia.org/schedule" id="dhtml_menu-3501">Schedule</a></li>
```
Comment (anshumanv) [inline _site/index.html]: Please indent all the child elements of the container accordingly.

**Review by anshumanv — CHANGES_REQUESTED** [review]

**Comment by anshumanv** [comment]
It should go like
```
container
    ul
        li
   /ul
/container
```

Please follow @samyakbvs

**Comment by samyakbvs** [comment]
Okay

**Comment by anshumanv** [comment]
Also please don't change other lines of code apart from the one you're adding.

**Comment by anshumanv** [comment]
Thanks, the indentation loos fine now, can you please check on the colors? This doesn't look close to the header nav we have at - https://fossasia.org/

**Comment by samyakbvs** [comment]
Just the colours or something else too?

**Comment by samyakbvs** [comment]
@anshumanv

**Comment by anshumanv** [comment]
Thanks for the changes, this can be merged as a WIP, let me ask others too.

**Review by anshumanv — APPROVED** [review]

**Review by the-codingninja — CHANGES_REQUESTED** [review]
Resolve conflicts

**Comment by samyakbvs** [comment]
now can it merge

**Comment by samyakbvs** [comment]
@ms10398

**Review by the-codingninja — APPROVED** [review]

**Comment by samyakbvs** [comment]
Now can it merge?

**Comment by samyakbvs** [comment]
@ms10398 and @anshumanv

**Comment by samyakbvs** [comment]
Please approve my task now @anshumanv and @ms10398

**Comment by mariobehling** [comment]
Not seeing improvements on the site.

File: _site/index.html [hunk 3]
```diff
@@ -15,10 +15,12 @@
     <link href='http://fonts.googleapis.com/css?family=Lobster' rel='stylesheet' type='text/css'>
     <link href='http://fonts.googleapis.com/css?family=Droid+Sans' rel='stylesheet' type='text/css'>
 	<link href="css/style.css" rel="stylesheet" type="text/css">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
```
Comment (mariobehling) [inline _site/index.html:18]: What do we need this for? Avoid additional dependencies.

**Pull request merged**